src/tests/store/activities/selectors.test.js

```diff
@@ -0,0 +1,23 @@
+import {
+  getActivitiesForDay,
+  getActivityItems,
+  getEditedActivity,
+} from '../../../store/activities/selectors';
+import { MONDAY } from '../../../constants/dates';
+import testProps from '../../fixtures/props';
+import testState from '../../fixtures/state';
+
+test('should return activities for given day', () => {
+  const result = getActivitiesForDay(testState.activities.items, MONDAY);
```

Comment:
I have some doubts if this selector is proper. Selector should receive state as argument, not items.